packages/common/src/core/traits/WithSlug.ts

```diff
@@ -13,5 +13,10 @@ export interface IWithSlug {
    * Organization information used to construct the
    * slug
    */
-  org: { id: string; key: string };
+  org?: {
```

Comment:
I'm open to other options for this... like just having orgUrlKey as a prop, and storing that in `item.properties.orgUrlKey` or something like that.